apps/web/src/components/StarRating.svelte

```diff
@@ -1,21 +1,21 @@
 <script lang="ts">
-  import type { Rating } from "./types";
-  export let rating: Rating = 0;
+
+  export let rating: number = 3;
 </script>
 
-<div class="rating">
-  {#each [1, 2, 3, 4, 5] as number}
-    <label>
-      <p class="h-0 opacity-0">{number}</p>
-      <input
-        type="radio"
-        name="rating"
-        class="mask mask-star cursor-auto bg-orange-400"
-        value={number}
-        bind:group={rating}
-        readonly
-        disabled
-      />
-    </label>
-  {/each}
-</div>
+<label class="leading-none">
+  <div class="rating relative">
+    {#each [1, 2, 3, 4, 5] as n}
+        <p class="h-0 opacity-0 absolute">{n}</p>
+        <input
+          type="radio"
+          name="rating"
+          class="mask mask-star cursor-auto bg-orange-400"
+          value={n}
+          bind:group={rating}
+          readonly
+          disabled
+        />
+    {/each}
+  </div>
+</label>
```

Comment:
Probably this is just because the formatter was run that this shows up in the diff

Reply:
I changed the variable "number" to n, and we had to put the label on the outside to fix something which enabled the stars to actually render its intended rating I think.